src/main/java/gov/nasa/jpl/mbee/mdk/options/MDKOptionsGroup.java

```diff
@@ -135,6 +139,29 @@ public void setUserScriptDirectory(String path) {
         addProperty(property, true);
     }
 
+    public List<String> getAuthenticationChain() {
+        String val = PROPERTY_RESOURCE_PROVIDER.getString(MMS_AUTHENTICATION_CHAIN, null);
+        if(val == null || val.isEmpty()) {
+            Property p = getProperty(MMS_AUTHENTICATION_CHAIN);
+            val = p.getValueStringRepresentation();
+        }
+        if (val == null || val.isEmpty()) {
+            return new ArrayList<>();
+        }
+        List<String> authChain = new ArrayList<String>();
+        for (String chainClass : Arrays.asList(val.split(","))) {
+            authChain.add(chainClass);
+        }
+        return authChain;
+    }
+
+    public void setAuthenticationChain(String value) {
+        StringProperty property = new StringProperty(MMS_AUTHENTICATION_CHAIN, value);
+        property.setResourceProvider(PROPERTY_RESOURCE_PROVIDER);
+        property.setGroup(GROUP);
+        addProperty(property, true);
```

Comment:
Do we want this property to be visible/editable to end-users? If not we should `visible=false`. If so we should add a description.

Reply:
Will update to false